pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -47,7 +47,7 @@ import (
 )
 
 // AdmissionPlugins is the full list of admission control plugins to enable in the order they must run
-var AdmissionPlugins = []string{"RunOnceDuration", lifecycle.PluginName, "PodNodeConstraints", "OriginPodNodeEnvironment", overrideapi.PluginName, serviceadmit.ExternalIPPluginName, "LimitRanger", "ServiceAccount", "SecurityContextConstraint", "BuildDefaults", "BuildOverrides", "ResourceQuota", "SCCExecRestrictions"}
+var AdmissionPlugins = []string{"RunOnceDuration", lifecycle.PluginName, "PodNodeConstraints", "OriginPodNodeEnvironment", overrideapi.PluginName, serviceadmit.ExternalIPPluginName, "LimitRanger", "ServiceAccount", "SecurityContextConstraint", "BuildDefaults", "BuildOverrides", "ResourceQuota", "SCCExecRestrictions", "ClusterResourceQuota"}
```

Comment:
I'm not sure which order will be better.  Doing it that way would increase
the chances of needing to compensate.

On Fri, May 13, 2016 at 3:19 PM, Dan Mace <notifications@github.com> wrote:

> In pkg/cmd/server/kubernetes/master_config.go
> <https://github.com/openshift/origin/pull/8686#discussion_r63237350>:
>
> > @@ -47,7 +47,7 @@ import (
> >  )
> >
> >  // AdmissionPlugins is the full list of admission control plugins to enable in the order they must run
> > -var AdmissionPlugins = []string{"RunOnceDuration", lifecycle.PluginName, "PodNodeConstraints", "OriginPodNodeEnvironment", overrideapi.PluginName, serviceadmit.ExternalIPPluginName, "LimitRanger", "ServiceAccount", "SecurityContextConstraint", "BuildDefaults", "BuildOverrides", "ResourceQuota", "SCCExecRestrictions"}
> > +var AdmissionPlugins = []string{"RunOnceDuration", lifecycle.PluginName, "PodNodeConstraints", "OriginPodNodeEnvironment", overrideapi.PluginName, serviceadmit.ExternalIPPluginName, "LimitRanger", "ServiceAccount", "SecurityContextConstraint", "BuildDefaults", "BuildOverrides", "ResourceQuota", "SCCExecRestrictions", "ClusterResourceQuota"}
>
> Worth noting that ClusterResourceQuota must come after ResourceQuota?
> (correct me if I'm wrong)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8686/files/ae54f9e67b867c4f4e12ba9c0094fe98c12c8789#r63237350>
>